feature/aft/aft_base/otg_tests/afts_base/README.md

```diff
@@ -109,7 +103,7 @@ Bring up both link between ATE:port1,port2 and DUT:port1,port2 using OTG api.
 
 *   BGP routes advertised from ATE:port1,port2 must have 2 next hops.
```

Comment:
For each of these cases, please can you add some text describing what we are testing. Why is this test needed vs. the steady state one? Are we checking that flapping a link doesn't cause route loss?

Reply:
this is explained in verification section.